sunpy/lightcurve/sources/soho.py

```diff
@@ -0,0 +1,164 @@
+
+    Examples
+    --------
+
+    .. plot::
+
+    
+        import os
+        import sunpy.data.test
+        filepath = sunpy.data.test.rootdir
+        from sunpy import lightcurve as lc
+        [header,data] = lc.ERNELightCurve._parse_txt(os.path.join(filepath , 'erne/', 'cr1907a.txt'))
+        erne = lc.ERNELightCurve(data,header)
+        erne.peek()
+
+    """
+
+    def peek(self, title="ERNE Two-hour averaged intensities"):
+        """Plots ERNE light curve in the usual manner"""
+
+        figure = plt.figure()
+        ax = plt.gca()
+
+        timerange_start = self.data['TimeRange'].apply(lambda col: col.start)
+        dates = matplotlib.dates.date2num(timerange_start.astype(datetime))
+
+        colors = ['Green','Red','Chocolate', 'Blue','SeaGreen','Tomato',
+                            'SlateBlue','Orange','Purple','Magenta','MediumVioletRed']
+        figure.delaxes(ax)
+        axes = figure.add_axes([0.1, 0.15, 0.55, 0.8])
+
+        for i,line in enumerate(self.header.values()):
+            if i >= 1:
+                axes.plot_date(dates, self.data[line].ffill(), '-',
+                     label=line[line.index('l')+2:], color=colors[i], lw=1)
+        
+        axes.set_yscale("log",nonposy='mask')
+        axes.set_title(title)
+        axes.set_ylabel('1/(cm^2*sr*s*MeV) [per nucleon in case of protons]')
+        axes.set_xlabel('UTC Time')
+
+        axes.yaxis.grid(True, 'major')
+        axes.xaxis.grid(False, 'major')
+        axes.legend(loc='center left', bbox_to_anchor=(1, 0.5))
+
+        figure.autofmt_xdate()
+        plt.show()
+
+        return figure
+
+    @staticmethod
+    def _parse_txt(filepath):
+        """
+        Parses a SOHO/ERNE file from
+        http://srl.utu.fi/erne_data/carrot/carrota.html
+        http://srl.utu.fi/erne_data/carrot/carrotp.html
+
+        and returns header as a list and ERNE data as pandas dataframe.
+        """
+        
+        #Reading in Data along with header
+        data = ascii.read(filepath, delimiter = "\s", data_start = 2 ) 
+
+        #header is read along with data i.e the first row in data is the header
+        #So extracting the first row elements and forming header list that contains the energy bins
+        header = [data[0][key] for key in data.colnames]
+        #and now excluding the first line to have only the data in data variable
+        data = data[1:]
+
+        data_modify = []
+
+        #Storing data columns in recognizable variables
+        year_col = data['col1']
+        month_col = data['col2']
+        date_col = data['col3']
+        begin_time_col = data['col4']
+        end_time_col = data['col5']
+
+        #Converting separate datetime element into a single datetime.datetime column
+        for i in range(len(data)):
+            date = datetime.strptime(year_col[i] + '-' + month_col[i] + '-' + date_col[i],"%y-%m-%d")
+            date1 = datetime.combine(date, time(int(begin_time_col[i][:2])))               #start time
```

Comment:
This is hardcoded to read just the hours... when the file also contain the minutes... what if one file has minutes as well?